gap/TGQuotientSequences.gd

```diff
@@ -70,6 +194,60 @@ DeclareGlobalFunction( "ExtendTGQuotientSequencesWithMinimalIndex" );
 #!
 #! @Section Export and Import
 
+#! @BeginGroup ExportTGQuotientSequencesAdjacencyMatrix
+#! @GroupTitle Exporting TGQuotientSequencesAdjacencyMatrix Objects
+#!
+#! @Description
+#!   Export the adjacency matrix `TGQuotientSequencesAdjacencyMatrix` <A>tgQSAdjMat</A> to the `OutputTextStream`  
+#!   <A>output-stream</A>
+#! @Arguments tgQSAdjMat, output-stream
+#! @Label for TGQuotientSequencesAdjacencyMatrix, OutputTextStream
+DeclareOperation( "Export", [ IsTGQuotientSequencesAdjacencyMatrixObj, IsOutputTextStream ] );
+#!
+#! @Description
+#!   or to the file at the path given by the string <A>path</A>.
+#! @Arguments tgQSAdjMat, path
+#! @Label for TGQuotientSequencesAdjacencyMatrix, String
+DeclareOperation( "Export", [ IsTGQuotientSequencesAdjacencyMatrixObj, IsString ] );
+#!
+#! @Description
+#!   Alterantively, the adjacency matrix can be exported to a string with
+#!   `ExportString`, which returns said string.
+#! @Arguments tgQSAdjMat
+#! @Label for TGQuotientSequencesAdjacencyMatrix
+DeclareOperation( "ExportString", [ IsTGQuotientSequencesAdjacencyMatrixObj ] );
+#!
+#! @EndGroup
+
+#!
+#! @BeginGroup ImportTGQuotientSequencesAdjacencyMatrix
+#! @GroupTitle Importing TGQuotientSequencesAdjacencyMatrix Objects
+#!
+#! @Description
+#!   Import an adjacency matrix from the `InputTextStream` <A>input-stream</A>
+#! @Arguments input-stream
+#! @Returns adjacency matrix as `TGQuotientSequencesAdjacencyMatrix`
+#! (see <Ref Sect='Section_TGQuotientSequencesAdjacencyMatrix'/>).
```

Comment:
This reference cannot be resolved, see suggested change at the beginning of the file.